rust/worker/src/system/scheduler.rs

```diff
@@ -71,6 +67,7 @@ impl Scheduler {
     /// Schedule a message to be sent to the component at a regular interval.
     ///
     /// `span_factory` is called immediately before sending the scheduled message to the component.
+    #[cfg(test)]
```

Comment:
This is not meant to be test-only exposed, its library code that should be available to use